### DISCUSSION TITLE
dongshanpiseven_standard-lvgl_defconfig

### DISCUSSION THREAD

**Discussion by DongshanPI** [DongshanPI/buildroot-external-dongshanpiseven#2] [General]
东山PI柒号 针对于LVGL-8.x 适配的标准系统配置，此页面是问题 建议专用讨论区。

**Comment by yangchao315** [comment]
使用dongshanpiseven_standard-lvgl_defconfig编译遇到错误：
version `GLIBC_2.29' not found
这个怎么解决？需要使用更高版本的ubuntu吗

**Comment by yangchao315** [comment]
https://github.com/DongshanPI/buildroot-external-dongshanpiseven/releases/  获取的LVGL镜像不显示，确认镜像是okay的吗
